server/api/impl/store/apps.go

```diff
@@ -57,7 +57,7 @@ func (s AppStore) Save(app *apps.App) error {
 	app.AppID = app.Manifest.AppID
 	app.Manifest = nil
 
-	conf.Apps[app.Manifest.AppID] = app.ConfigMap()
+	conf.Apps[app.AppID] = app.ConfigMap()
```

Comment:
Not strictly related to this PR: Why do we store the `AppID` both in `App` and in `App.Manfest`. Are there Apps without a `Manifest`?

Reply:
Good question. Apps are stored in the config. Manifests are stored in memory. We need appID to "connect" them. Manifests are not stored in config because of the size.
What that means:
Synchronization of the `apps.json` is run in every done of the HA cluster, so manifests as well as apps are synced.
But whenever we install a non-`apps.json` app from a slash command, manifest will be saved only on one node. And in the HA environment it will not work. I suppose that path is for testing and developer story only anyway and should not be a big problem.
cc @levb

Reply:
First of all, thanks for the explanation :+1: 

Can we rely for non-`apps.json` apps i.e. HTTP apps (correct?) to fetch that data from the app ifself via HTTP and then cache in memory?

Reply:
We can, but that's not implemented yet. And we have to do that on every HA node.